csr2transmart/mappers/ontology_mapper.py

```diff
@@ -14,6 +14,7 @@ class OntologyMapper:
     def __init__(self, top_tree_node: str):
         self.top_tree_node = top_tree_node
         self.concept_code_to_concept: Dict[str, Concept] = {}
+        self.subject_entity_names = list(map(lambda se: se.schema()['title'], SubjectEntity.__args__))
```

Comment:
Could even be above the class, it is not depending on anything of `self`, but this also works.